recordm/customUI/dash/src/components/Calendar.vue

```diff
@@ -36,8 +36,57 @@
 
   const DEFAULT_EVENT_COLOR = '#0e7bbe'
   const MAX_VISIBLE_DAY_EVENTS = 3
+  const defaultHeaderToolbar = {
+    left: 'today prev next',
+    center: '',
+    right: 'dayGridWeek,dayGridMonth,listMonth,multiMonthYear,listYear'
+  }
+  const listYearToolBar = {
+    left: `today prev next`,
+    center: '',
+    right: 'dayGridWeek,dayGridMonth,listMonth,multiMonthYear,listYear'
+  }
+  function getInputTypeOfDate(date){
+    let monthDay = date.getDate() < 10 ? `0${date.getDate()}` :  date.getDate()
+    let month = date.getMonth()+1 < 10 ? `0${date.getMonth()+1}` :  date.getMonth()+1
+    return `${date.getFullYear()}-${month}-${monthDay}` 
+  }
```

Comment:
Não estou certo disto. Pelo que entendi estás a a tentar formatar as datas. certo ? O que é que acontece se o teu locale for US ou mesmo ES? Já testaste?

Reply:
Não vai haver problemas:

<img width="743" alt="image" src="https://github.com/user-attachments/assets/243dffed-fd79-455a-a6d2-acf88eb57ac8">

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/date